packages/orchestration/src/model-types.ts

```diff
@@ -4,44 +4,39 @@ type LiteralUnion<T extends U, U = string> = T | (U & Record<never, never>);
  * Azure OpenAI models for chat completion.
  */
 export type AzureOpenAiChatModel = LiteralUnion<
-  | 'gpt-4o-mini'
-  | 'gpt-4o'
-  | 'gpt-4'
-  | 'gpt-4-32k'
-  | 'gpt-35-turbo'
-  | 'gpt-35-turbo-0125'
-  | 'gpt-35-turbo-16k'
->;
-
-/**
- * Azure OpenAI models for embedding.
- */
-export type AzureOpenAiEmbeddingModel = LiteralUnion<
-  'text-embedding-ada-002' | 'text-embedding-3-small' | 'text-embedding-3-large'
+  'gpt-4o' | 'gpt-4' | 'gpt-4-32k' | 'gpt-35-turbo' | 'gpt-35-turbo-16k'
 >;
 
 /**
  * GCP Vertex AI models for chat completion.
  */
 export type GcpVertexAiChatModel = LiteralUnion<
-  'gemini-1.0-pro' | 'gemini-1.5-pro' | 'gemini-1.5-flash' | 'chat-bison'
+  'gemini-1.0-pro' | 'gemini-1.5-pro' | 'gemini-1.5-flash'
 >;
 
 /**
  * AWS Bedrock models for chat completion.
  */
 export type AwsBedrockChatModel = LiteralUnion<
-  | 'amazon--titan-text-express'
-  | 'amazon--titan-text-lite'
   | 'anthropic--claude-3-haiku'
   | 'anthropic--claude-3-opus'
   | 'anthropic--claude-3-sonnet'
   | 'anthropic--claude-3.5-sonnet'
 >;
 
 /**
- * All available models for chat completion.
+ * Aicore Opensource models for chat completion.
+ */
+export type AicoreOpensourceChatModel = LiteralUnion<
+  'mistralai--mixtral-8x7b-instruct-v01' | 'meta--llama3-70b-instruct'
+>;
+
+/**
+ * Supported chat models for orchestration.
  */
 export type ChatModel = LiteralUnion<
```

Comment:
[q] Why do we need to wrap this again in LiteralUnion?

Reply:
Oh I forgot to remove this. I thought about leaving LiteralUnion out of the reused model-types in `core` at first. Removed.